tests/e2e/swarm_test.go

```diff
@@ -235,7 +235,7 @@ func (s *SwarmTestSuite) TestFailoverAcrossReplicas(c *C) {
 	status, host = verification.IsDockerContainerRunning(s.swarmNodes, s.serviceName, 2)
 	c.Assert(status, Equals, true, Commentf("Container %s is not running", s.serviceName))
 
-	status = verification.VerifyAttachedStatus(s.volumeName, host, s.esxName)
+	status = verification.VerifyAttachedStatusByIP(s.volumeName, host, s.esxName)
```

Comment:
I see this util being used only in this swarm test suite. Is it because the commonly populated test par (usually referred as s.config in other tests) isn't available here?

Else we could have gone with just one VerifyAttachedStatus and passing expected name to it